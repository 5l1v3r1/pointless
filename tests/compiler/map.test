
y = {1: "", 2: false}
x = {}