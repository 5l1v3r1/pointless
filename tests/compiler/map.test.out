   0 [ SaveImports  ] 0       
   1 [ MakeEnv      ] 2       
   2 [ MakeNumber   ] 6       
   3 [ MakeDef      ] 0       (y)
   4 [ Jump         ] 8       (-> 12)
   5 [ MakeNumber   ] 1       
   6 [ LoadStr      ] 8       ("")
   7 [ MakeNumber   ] 2       
   8 [ LoadBool     ] 0       
   9 [ MakeMap      ] 2       
  10 [ SaveVal      ] 0       (y)
  11 [ Return       ]
  12 [ MakeNumber   ] 9       
  13 [ MakeDef      ] 1       (x)
  14 [ Jump         ] 4       (-> 18)
  15 [ MakeMap      ] 0       
  16 [ SaveVal      ] 1       (x)
  17 [ Return       ]
  18 [ MakeObj      ]
  19 [ FieldRef     ] 11      (output)
  20 [ Jump         ] 6       (-> 26)
  21 [ Dup          ]
  22 [ LangFieldRef ] 18      (!getHead)
  23 [ Pop          ] 1       
  24 [ Jump         ] 2       (-> 26)
  25 [ LangFieldRef ] 27      (!getTail)
  26 [ Dispatch     ]
